sdk/core/azure-core-experimental/azure/core/experimental/transport/_httpx.py

```diff
@@ -66,7 +94,9 @@ class HttpXStreamDownloadGenerator:
     """Generator for streaming response data.
 
     :param pipeline: The pipeline object
+    :type pipeline: ~azure.core.pipeline.Pipeline
     :param response: The response object.
+    :type response: ~azure.core.rest.HttpResponse
```

Comment:
Do we need to update this too?

Reply:
no as discussed, the response will only be rest.HttpResponse